packages/realm-web/src/User.ts

```diff
@@ -43,14 +40,21 @@ type JWT<CustomDataType extends object = any> = {
     userData: CustomDataType;
 };
 
+/** The state of a user within the app */
 export enum UserState {
+    /** Active, with both access and refresh tokens */
     Active = "active",
+    /** Logged out, but there might still be data persisted about the user, in the browser. */
     LoggedOut = "logged-out",
+    /** Logged out and all data about the user has been removed. */
     Removed = "removed",
 }
 
+/** The type of a user. */
 export enum UserType {
```

Comment:
Maybe it isn't worth changing this at this point, but I usually prefer using the word "Kind" in contexts like this rather than "Type" to avoid ambiguity with types in the type system.

Reply:
Yeah - I'm just following the naming from the server here: The server returns this in a property named `type` on the users profile response.